test/error.js

```diff
@@ -14,21 +13,20 @@ test('test error', function (t) {
     pull.values(['x', 'y', 'z']),
     pull.through(null, function (err) {
       if(_err) {
-        t.strictEqual(err, _err);
+        t.deepEqual(err, _err);
         t.end();
       }
       _err = err
     }),
     ws(new WebSocket('ws://localhost:34897/' + Math.random())),
     pull.collect(function (err) {
       if(_err) {
-        t.strictEqual(err, _err);
```

Comment:
`t.strictEqual` is an alias of `t.equal` which checks `===`  .. this no longer passes.

looks like the `err` is an object so presume using `deepEqual` (which checks `===` on each of the leaves) is fine.

Reply:
this is intended to check that it is exactly the same error object, not a copy.